docs/ref/models/indexes.txt

```diff
@@ -46,3 +45,15 @@ than 30 characters and shouldn't start with a number (0-9) or underscore (_).
     Use the :class:`~django.db.migrations.operations.AddIndex` and
     :class:`~django.db.migrations.operations.RemoveIndex` operations to add
     and remove indexes.
+
+Built-in indexes
+================
+
+.. class:: Index(fields=[], name=None)
+
+    Creates the default index (B-Tree) in the database.
+
+.. class:: Hash(Index)
+
+    Creates a hash index in the database. This index is supported only by
+    postgresql.
```

Comment:
I was not sure about that so I had asked in the ticket, but that's still to be accepted.